pkg/devspace/deploy/deployer/helm/deploy.go

```diff
@@ -44,6 +45,14 @@ func (d *DeployConfig) Deploy(ctx devspacecontext.Context, forceDeploy bool) (bo
 		releaseNamespace = d.DeploymentConfig.Namespace
 	}
 
+	if d.DeploymentConfig.Helm.Chart.Source != nil {
+		downloadPath, err := d.Helm.DownloadChart(ctx, d.DeploymentConfig.Helm)
+		if err != nil {
+			return false, errors.Wrap(err, "download chart")
+		}
+		chartPath = downloadPath
+	}
```

Comment:
This adds overhead to each to deploy, but hopefully better than redeploying each time.